### ISSUE TITLE
Duplicate Organisations

### ISSUE THREAD

**Issue by PhilRumens** [Jumoo/Taskily#2]
On sign-up a new organisation is created even if you specify one with the exact same name as a previous sign-up

![taskily](https://cloud.githubusercontent.com/assets/4989597/10336881/b25d3adc-6cf3-11e5-8bb9-0d9a9ae5697b.png)

**Comment by KevinJump** [comment]
yeah, coming from the cloud app, you can have multiple orgs with the same name (stops someone else hijacking another account). 

However i suspect for local people will only want one. It's probably more a case of only creating the organisation for the first user. 

I will look at adding some better logic for a local install to the account code, but if you want a quick fix, you could change the code in the Register here https://github.com/Jumoo/Taskily/blob/master/TaskilyWeb/Controllers/AccountController.cs#L108 
there is no reason in the model why an organisation can't have multiple users (just setting the org id to the same for all users.) 
- you could just assign a user to an existing one. 
- you could also set the org to standard then you get all the limits and stuff taken off. 
- (and you could choose not to have the demo survey).

I'll leave this issue open until i get around to changing the code .

**Comment by PhilRumens** [comment]
Sure, I'll add all new users to an existing org and comment out the demo survey.
